.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -110,10 +112,14 @@ jobs:
         run: |
           .circleci/scripts/windows_cudnn_install.sh
 {%- endif %}
+      !{{ common.parse_ref() }}
       - name: Build
         shell: bash
         env:
           PYTORCH_FINAL_PACKAGE_DIR: /c/${{ github.run_id }}/build-results/
+          CIRCLE_BRANCH: ${{ steps.parse-ref.outputs.branch }}
+          CIRCLE_PR_NUMBER: ${{ github.event.pull_request.number }}
+          CIRCLE_SHA1: ${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
Should these just be toplevel env at this point?

Reply:
This doesn't have to be this PR

Reply:
agreed, `CIRCLE_BRANCH` relies on an actual step though so we have to keep that one per-step